options.c

```diff
@@ -149,6 +152,20 @@ void parse_options(int argc, char **argv)
 					error(EXIT_FAILURE, 0, "Invalid argument for option -s: %s", optarg);
 				_options.scalemode = s - scalemodes;
 				break;
+			case 'T':
+				if (*optarg == ';') {
+					_options.title_prefix = ++optarg;
```

Comment:
Is optarg guaranteed to be null terminated?

Also if someone puts `-T ";a"` it should error out for invalid suffix, not set prefix as "a".  And `-T ";2"` should set prefix to be empty and suffix to be fullpath.

Reply:
It is guaranteed to be null-terminated.

It used to error out. I didn't think of this in the format reversal.